app/Http/Controllers/DataController.php

```diff
@@ -89,11 +90,11 @@ public function show(Request $request)
                         $manejo_parcelas = ManejoParcela::select()->whereIn('comunidad_id',$request->comunidadsSelected)->paginate(5);
 
                     }
-                    if($cultivo_modules=='plagas'){
+                    if($parcelas_modules=='plagas'){
```

Comment:
Are plagas and enfermedades plot-level modules or crop-level modules? I don't think they can be both? (I thought they were crop-level, as they appear in the crop-level repeat section in the form.).

Reply:
This is for the Control Panel on the main page. The user can looks at the data from the point of view of crops or the plots and they can select the models they are interested in.

Reply:
Yes, but my point is about the list of modules you have included under each of plots and crops. I’m curious as to why you have plot management, pests, diseases and yield under both plot and crop modules, and soils under only plots and phenology under only crops. Not saying it’s wrong - You may have a good reason that I don’t know about (as I don’t know the structures that well!) 

I think this is something to review as part of going through the front-end / data extraction section later on in October :)